Makefile

```diff
@@ -1,6 +1,6 @@
 data/whitelisted_ext_idefs.json: 
 	python3 wikidatarefisland/run.py --step ss1 --output "whitelisted_ext_idefs.json"
-data/pipe1.json:
-    python3 wikidatarefisland/run.py --step itemextract --input "dump.json" --output "pipe1.jsonl"
+data/extracted_unreferenced_statements.jsonl:
+    python3 wikidatarefisland/run.py --step itemextract --input "dump.json" --output "extracted_unreferenced_statements.jsonl"
 data/scraped_data.json:
```

Comment:
smae here, we should add `extracted_unreferenced_statements.jsonl` as a dependency to the scrape step (we missed that on the PR foe pipe 2)

Reply:
cool! done!